src/variables/DynPoint2D.jl

```diff
@@ -11,3 +11,5 @@ mutable struct DynPoint2 <: IncrementalInference.InferenceVariable
   manifolds::Tuple{Symbol, Symbol, Symbol, Symbol}
   DynPoint2(;ut::Int64=-9999999999) = new(ut, 4,(:Euclid,:Euclid,:Euclid,:Euclid,))
 end
+getDimension(::DynPoint2) = 4
```

Comment:
can also consider avoiding the object allocation with `getDimension(::Type{DynPoint2}) = 4`

Reply:
We can consider that at a later stage. For now, the softtype field still exists for compatibility with types such as DynPose2, so instances are passed.